Hackney.Shared.HousingSearch/Gateways/Models/Tenures/QueryableTemporaryAccommodationInfo.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using Nest;
+
+namespace Hackney.Shared.HousingSearch.Gateways.Models.Tenures
+{
+    public class QueryableTemporaryAccommodationInfo
+    {
+        public string BookingStatus { get; set; }
```

Comment:
Are we content with adding in just the `BookingStatus`?
Any chance we'd want to search by assigned officer's `Email` _(say we want to get all the 'assigned to me' bookings)_ ?

Reply:
@Duslerke We'll definitely need the officer details and that's something we'll work on next as part of the assigned to & search filter piece. This is just he bare minimum for the specific requirement we are working on now.